app-crypt/acme-sh/acme-sh-2.7.9.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-/.}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="An ACME Shell script"
+HOMEPAGE="https://github.com/Neilpang/acme.sh"
+SRC_URI="https://github.com/Neilpang/${MY_PN}/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="apache2 +cron nginx libressl standalone"
+
+RDEPEND="net-misc/curl
+	|| ( net-analyzer/netcat6 net-analyzer/openbsd-netcat )
+	sys-apps/iproute2
+	sys-apps/net-tools
+	sys-apps/util-linux
+	apache2? ( www-servers/apache:2 )
+	cron? ( virtual/cron )
+	libressl? ( dev-libs/libressl )
```

Comment:
sub-slot operator?

Reply:
@Zlogene, it's a shell script.

Reply:
@mgorny ugh, I was way too fast and forgot to remove this nitpick:p